packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -9,42 +9,70 @@
 URL:        https://github.com/theforeman/foreman_remote_execution
 Source0:    https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Requires:   foreman >= 1.20.0
+Requires:   foreman >= 1.24.0
+
+BuildRequires: systemd
 
+# start specfile generated dependencies
+Requires: foreman >= %{foreman_min_version}
 Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}rubygems
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(deface)
 Requires: %{?scl_prefix}rubygem(dynflow) >= 1.0.1
 Requires: %{?scl_prefix}rubygem(dynflow) < 2.0.0
+Requires: %{?scl_prefix}rubygem(foreman_remote_execution_core)
 Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.15.1
 Requires: %{?scl_prefix}rubygem(foreman-tasks) < 1.0.0
-Requires: %{?scl_prefix}rubygem(foreman_remote_execution_core)
-Requires: %{?scl_prefix}rubygem(deface)
-
-BuildRequires: foreman-plugin >= 1.20.0
-BuildRequires: foreman-assets
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
-BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: foreman-assets >= %{foreman_min_version}
+BuildRequires: foreman-plugin >= %{foreman_min_version}
+BuildRequires: %{?scl_prefix}rubygem(deface)
 BuildRequires: %{?scl_prefix}rubygem(dynflow) >= 1.0.1
 BuildRequires: %{?scl_prefix}rubygem(dynflow) < 2.0.0
+BuildRequires: %{?scl_prefix}rubygem(foreman_remote_execution_core)
 BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.15.1
 BuildRequires: %{?scl_prefix}rubygem(foreman-tasks) < 1.0.0
-BuildRequires: %{?scl_prefix}rubygem(foreman_remote_execution_core)
-BuildRequires: %{?scl_prefix}rubygem(deface)
-BuildRequires: systemd
-
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}ruby
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
 BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+Provides: foreman-plugin-%{plugin_name} = %{version}
+# end specfile generated dependencies
+
+# start package.json devDependencies BuildRequires
+BuildRequires: npm(babel-plugin-lodash) >= 3.3.2
+BuildRequires: npm(babel-plugin-lodash) < 4.0.0
+BuildRequires: npm(babel-plugin-transform-class-properties) >= 6.24.1
+BuildRequires: npm(babel-plugin-transform-class-properties) < 7.0.0
+BuildRequires: npm(babel-plugin-transform-object-assign) >= 6.22.0
+BuildRequires: npm(babel-plugin-transform-object-assign) < 7.0.0
+BuildRequires: npm(babel-plugin-transform-object-rest-spread) >= 6.26.0
+BuildRequires: npm(babel-plugin-transform-object-rest-spread) < 7.0.0
+BuildRequires: npm(babel-preset-env) >= 1.6.0
+BuildRequires: npm(babel-preset-env) < 2.0.0
+BuildRequires: npm(babel-preset-react) >= 6.24.1
+BuildRequires: npm(babel-preset-react) < 7.0.0
+# end package.json devDependencies BuildRequires
+
+# start package.json dependencies BuildRequires
+BuildRequires: npm(@theforeman/vendor) >= 1.4.0
+BuildRequires: npm(@theforeman/vendor) < 2.0.0
```

Comment:
All of the `npm(` calls will need updating to `%{?scl_prefix}npm(` due to the chnage over to using NodeJS SCL in our builds.

Reply:
Just like that?